src/storage/v2/schemas.hpp

```diff
@@ -73,48 +73,52 @@ class Schemas {
   Schemas &operator=(Schemas &&) = delete;
   ~Schemas() = default;
 
-  [[nodiscard]] bool CreateSchema(LabelId label, const std::vector<SchemaProperty> &schemas_types);
+  [[nodiscard]] SchemasList ListSchemas() const;
 
-  [[nodiscard]] bool DeleteSchema(LabelId label);
+  [[nodiscard]] SchemasList GetSchema(LabelId primary_label) const;
```

Comment:
Don't return `SchemaList`, that is an interface lie, return a single schema, and the caller should create a list if he wants to